ui/app/js/controllers/user_edit_controller.js

```diff
@@ -148,7 +152,7 @@ function ($scope, $modalInstance, CSRF, Permissions, user, users) {
       })
       .error(function(response) {
         if (typeof response === 'object') {
-          $scope.errors = response;
+          $scope.errors.permission = response;
```

Comment:
After rereading some other code, I'm not sure this is the right thing to do. It ends up with keys like: errors.permission.permission and errors.permission.options, but the templates end up looking for things like errors.options.